### PR TITLE
Change bitwise queries and remove unsupported operations

### LINKED ISSUE #203: Query generated using bitwise operations has error in Doctrine
I was using this lib and found an error while using it with doctrine. When I do some bitwise spec, the generate query looks like this `field & value`, but doctrine doesn't support this format, in fact it should be like `BIT_AND(...)`.

My workaround to solve it was to use `Spec::fun('BIT_AND', [field, value])`, this error only occurs using Spec with doctrine. A possible solution is to change the operation and format the query that is generated by the Spec. If you want, I can make a pull request.

### PR DESCRIPTION
**Pull request by ViniciusMartinsSouza**
fix #203

### CODE REVIEW

**Comment by peter-gribanov** [comment]
Thank you for your work, but we cannot delete these functions now.
This change violates backward compatibility and requires a major release.
It is necessary to fix the problem without disturbing the public interface.

File: src/Operand/Bitwise.php [hunk 1]
```diff
@@ -7,25 +7,16 @@
...
+    const B_AND = 'BIT_AND';
 
-    const B_OR = '|';
-
-    const B_XOR = '^';
-
-    const B_LS = '<<';
-
-    const B_RS = '>>';
+    const B_OR = 'BIT_OR';
```
Comment (peter-gribanov) [inline src/Operand/Bitwise.php]: These are public constants. Do not change their meaning. This breaks BC.

File: src/Spec.php [hunk 2]
```diff
@@ -666,49 +662,6 @@ public static function bOr($field, $value)
...
-    /**
-     * @param Operand|string $field
-     *
-     * @return BitNot
-     */
-    public static function bNot($field)
-    {
-        return new BitNot($field);
-    }
-
```
Comment (peter-gribanov) [inline src/Spec.php]: Do not delete broken methods. This breaks BC.

File: tests/Operand/BitRightShiftSpec.php [hunk 3]
```diff
@@ -1,51 +0,0 @@
...
-
-        $this->transform($qb, 'a')->shouldReturn('(a.foo >> :comparison_10)');
-    }
-
-    public function it_is_transformable_add_fields(QueryBuilder $qb)
-    {
-        $this->beConstructedWith(new Field('foo'), new Field('bar'));
-        $this->transform($qb, 'a')->shouldReturn('(a.foo >> a.bar)');
-    }
-}
```
Comment (peter-gribanov) [inline tests/Operand/BitRightShiftSpec.php]: These tests can be removed. :+1: They no longer make sense. Removing these tests will not break BC.

File: tests/SpecSpec.php [hunk 4]
```diff
@@ -70,26 +66,6 @@ public function it_create_bit_or_operand()
...
-
-    public function it_create_bit_right_shift_operand()
-    {
-        $this->bRs('foo', 'bar')->shouldReturnAnInstanceOf(BitRightShift::class);
-    }
-
-    public function it_create_bit_not_operand()
-    {
-        $this->bNot('foo')->shouldReturnAnInstanceOf(BitNot::class);
-    }
```
Comment (peter-gribanov) [inline tests/SpecSpec.php]: These tests can be removed. :+1: They no longer make sense. Removing these tests will not break BC.

**Comment by ViniciusMartinsSouza** [comment]
Thanks for your feedback, changes applied in the last commit. That needs to be changed, please let me know

**Comment by peter-gribanov** [comment]
All is good. Thanks

**Pull request merged**